src/go/migration-cli/tests/data/complex_cluster_001.yaml

```diff
@@ -0,0 +1,151 @@
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+  annotations:
+    redpanda.vectorized.io/managed: "true"
+  name: complex-cluster-001
+spec:
+  additionalConfiguration:
+    redpanda.aggregate_metrics: "true"
+    redpanda.cloud_storage_access_key: unused
+    redpanda.cloud_storage_enable_remote_read: "true"
+    redpanda.cloud_storage_enable_remote_write: "true"
+    redpanda.cloud_storage_secret_key: unused
+    redpanda.cloud_storage_segment_max_upload_interval_sec: "3600"
+    redpanda.cluster_id: rp-chhnf2abksi1fj6tastg
+    redpanda.compacted_log_segment_size: "67108864"
+    redpanda.default_topic_replications: "3"
+    redpanda.enable_rack_awareness: "true"
+    redpanda.group_topic_partitions: "16"
+    redpanda.internal_topic_replication_factor: "3"
+    redpanda.kafka_batch_max_bytes: "1048576"
+    redpanda.kafka_connection_rate_limit: "1000"
+    redpanda.kafka_connections_max: "15100"
+    redpanda.log_segment_size: "134217728"
+    redpanda.log_segment_size_max: "268435456"
+    redpanda.log_segment_size_min: "16777216"
+    redpanda.max_compacted_log_segment_size: "536870912"
+    redpanda.max_concurrent_producer_ids: "25000"
+    redpanda.partition_autobalancing_mode: continuous
+    redpanda.topic_partitions_per_shard: "1000"
```

Comment:
Why those are not passed to `.Values.config.cluster` and then asserted?

Reply:
Because I am not sure about how and if they should be passed, we can definitely make the change now. Some of those however belong in the tier storage fields right? If we do move them, to config.cluster should the "redpanda." prefixes be removed or not?

Reply:
> should the "redpanda." prefixes be removed or not?

The `redpanda.` prefix is used in operator to differentiate rpk config from Redpanda config. 

The prefix needs to be dropped in helm chart definition.